posthog-node/src/extensions/feature-flags/feature-flags.ts

```diff
@@ -470,19 +470,19 @@ class FeatureFlagsPoller {
 
         case 200: {
           // Process successful response
-          const responseJson = await res.json()
+          const responseJson = ((await res.json()) as { [key: string]: any }) ?? {}
```

Comment:
JsonType could be used here, it's parsing JSON afterall

Reply:
thats what i did initially but then i'd need to cast to a key: value thing anyway otherwise `'flags' in responseJson` wouldn't work